aioriak/transport.py

```diff
@@ -624,7 +625,9 @@ def _decode_quorum(self, rw):
         return responses
 
     async def _request(self, msg_code, msg=None, expect=None):
-        self._writer.write(self._encode_message(msg_code, msg))
+        if not isinstance(msg, Msg):
+            msg = self._encode_message(msg_code, msg)
+        self._writer.write(msg)
```

Comment:
I think we can't pass Msg instance directly into https://docs.python.org/3/library/asyncio-protocol.html#asyncio.WriteTransport.write.
We need to add some transformation method Msg() -> bytes

Reply:
Maby we can use something like:
```python
self._encode_message(*msg[:2])
```